src/main/resources/application.properties

```diff
@@ -130,6 +130,8 @@ azure.synapse.encryptionKey=
 azure.synapse.initialize=false
 azure.synapse.connectRetryInterval=15
 azure.synapse.connectRetryCount=20
+# Number of concurrent operations on Azure storage tables.  Note operations can be batches of operations.
+azure.threading.numTableThreads=1000
```

Comment:
How did you get to this number?

Reply:
I actually came across this from my adventures in batch file copy land.  1000 seems to be a big enough number where we shouldn't add a lot of memory / CPU pressure on the system